assets/queries/cloudFormation/iam_access_keys_active_for_root/test/positive.yaml

```diff
@@ -0,0 +1,6 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Resources:
+  CFNKeys:
+    Type: AWS::IAM::AccessKey
+    Properties:
+      UserName: !Ref Root
```

Comment:
[As a good POSIX practice](https://unix.stackexchange.com/a/18789) add end-of-file newline